Plugins/Mods/Flight Vector.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<PluginData xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="ModPlugin">
+  <Id>2950011596</Id>
+  <FriendlyName>Flight Vector</FriendlyName>
+  <Author>BDCarrillo</Author>
+  <Tooltip>Overview
+When seated in a cockpit, this mod draws lines to depict the prograde, retrograde, and gravity vector for your current grid. Note these lines are a fixed length, and only indicate direction.
+
```

Comment:
@austinvaness We can't find it in the list because it lacks the closing tag `</Tooltip>` & the opening tag `<Description>`

Reply:
This was fixed: https://github.com/sepluginloader/PluginHub/commit/4d9e606fa47d4e139592db3555a5271508edf887